tests/php/sync/test_class.jetpack-sync-options.php

```diff
@@ -56,6 +56,14 @@ public function test_sync_initalize_Jetpack_Sync_Action_on_init() {
 		$this->assertEquals( 11, has_action('init', array( 'Jetpack_Sync_Actions', 'init' ) ) );
 	}
 
+	public function test_sync_active_modules() {
+		// Instead of calling $this->client->set_defaults();
+		$this->client->set_options_whitelist( array( 'jetpack_active_modules' ) );
+		Jetpack_Options::update_option( 'active_modules', array( 'manage' ) );
+		$this->client->do_sync();
+		$this->assertContains( 'jetpack_active_modules', Jetpack_Sync_Defaults::$default_options_whitelist );
```

Comment:
This seems unnecessary for the test, since the line below it will always fail if this one does. It also introduces brittle coupling, as a change in the implementation of how we set defaults (e.g. maybe this uses the filter) would break the test, without breaking the functionality.